components/script/script_thread.rs

```diff
@@ -68,6 +68,7 @@ use js::rust::wrappers2::{JS_AddInterruptCallback, SetWindowProxyClass};
 use layout_api::{LayoutConfig, LayoutFactory, RestyleReason, ScriptThreadFactory};
 use media::WindowGLContext;
 use metrics::MAX_TASK_NS;
+use net::image_cache::ImageCacheFactoryImpl;
```

Comment:
Mmm. Maybe the image cache implementation should move into the script crate instead, if we now need this here.

Reply:
I went with your second suggestion. Eventually `ImageCacheImpl` should probably move entirely out of `net` into its own crate.